poc/ibm_api.py

```diff
@@ -0,0 +1,44 @@
+from qiskit import *
+from qiskit import IBMQ
+from client import *
+import os
+
+token=os.getenv('TOKEN')
+IBMQ.save_account(token)
+IBMQ.load_account() # Load account from disk
+print(IBMQ.providers())  # List all available providers
+
+provider = IBMQ.get_provider(hub='ibm-q')
+IBMQ.get_provider(group='open')
+print("The backends are:")
+print(provider.backends())
+
+print("Selecting backend ibmq_qasm_simulator...")
+backend = provider.get_backend('ibmq_qasm_simulator')
+print(backend)
+
+#Submitting a job
+
+job = execute(qc, backend)
```

Comment:
I would refrain from using the `execute`. From what I remember execute is asynchronous and the client calling execute waits for the job to be added on the backend queue. Use `transpile` and then `assemble` to get the JSON schema of the object.

```
qc = QuantumCircuit(2, 2)
qc.h(0)
qc.cx(0, 1)
qc.measure([0, 1], [0, 1])
backend_sim = BasicAer.get_backend('qasm_simulator')
transpiled_qc = transpile(qc, backend_sim)
qasm_obj = assemble(transpiled_qc)
```

Reply:
Follow this https://github.com/qiskit-community/openshift-quantum-operators/blob/master/operators-examples/openshift-ibm-quantum-operator/getting_started.md

Reply:
Sure @husky-parul , will make that change